ygot/path_types_test.go

```diff
@@ -21,14 +21,20 @@ import (
 	"google.golang.org/protobuf/proto"
 )
 
+type deviceRoot struct {
```

Comment:
why is this type necessary? Why can't you just use Device directly?

Reply:
I want to showcase that the intended usage is to allow embedding.